autocertkit/network_tests.py

```diff
@@ -678,16 +678,25 @@ def _run_test(self, session, direction):
             
         log.debug("About to run iperf test...")
 
-        #Run an iperf test - if failure, an exception should be raised.
-        iperf_data = IperfTest(session, client, server, 
+        #Run an iperf test - if failure (with 1 additional attempt), an exception should be raised.
+        try:
```

Comment:
Hi, this is fair approach. But for better extendability, can you reimplement using 'looping' with retry count?